backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -160,37 +163,24 @@ public LiveOktaRepository(
     return Optional.of(claims.get(0));
   }
 
-  public Map<String, OrganizationRoleClaims> getAllUsersForOrganization(Organization org) {
-
-    GroupList orgGroups =
-        _client.listGroups(generateGroupOrgPrefix(org.getExternalId()), null, null);
-    if (orgGroups.stream().count() == 0) {
-      throw new IllegalGraphqlArgumentException(
-          "Cannot get Okta users for org=" + org.getExternalId() + ": Okta groups are nonexistent");
-    }
-
-    // Create a map from each user to their Okta groups...
-    Map<String, Set<String>> usersToGroupNames = new HashMap<>();
-    orgGroups.forEach(
-        g -> {
-          String groupName = g.getProfile().getName();
-          g.listUsers()
-              .forEach(
-                  u -> {
-                    if (u.getStatus() != UserStatus.SUSPENDED) {
-                      String username = u.getProfile().getEmail();
-                      if (!usersToGroupNames.containsKey(username)) {
-                        usersToGroupNames.put(username, new HashSet<>());
-                      }
-                      usersToGroupNames.get(username).add(groupName);
-                    }
-                  });
-        });
-
-    // ...then convert each user's relevant Okta groups to an OrganizationRoleClaims object
-    return usersToGroupNames.entrySet().stream()
-        .collect(
-            Collectors.toMap(e -> e.getKey(), e -> _extractor.convertClaims(e.getValue()).get(0)));
+  public Set<String> getAllUsersForOrganization(Organization org) {
+    final String noAccessGroupName =
```

Comment:
super nitpick -- this variable really represents everyone in the org, not everyone with no access (even though it's looking for everyone in the `NO_ACCESS` group. might suggest renaming `noAccessGroupName` to `orgDefaultGroupName`